### PR TITLE
CSS Loader: add Glasscord compatibility

### PR DESCRIPTION
**Pull request by AryToNeX**
This commit adds [Glasscord](https://github.com/AryToNeX/Glasscord) integration to the CSS loader.

Glasscord is a tool for theme creators and users that enables the Discord window to be transparent without all the drawbacks that we're used to on Windows. Additional features include native compositing for Fluent Design's Blur on Windows and Vibrancy on macOS; all of this is manageable via simple CSS properties.

The project is only several days old, so if you have any question please ask me.

### CODE REVIEW

File: plugins/css_loader.js [hunk 1]
```diff
@@ -49,13 +50,19 @@ module.exports = new Plugin({
                 document.head.appendChild(window.customCss);
             }
             window.customCss.innerHTML = css;
+            if(window.glasscord)
```
Comment (rauenzi) [inline plugins/css_loader.js:53]: This one makes sense if you are having issues with load order.

File: plugins/css_loader.js [hunk 2]
```diff
@@ -49,13 +50,19 @@ module.exports = new Plugin({
...
 
             if (window.cssWatcher == null) {
                 window.cssWatcher = fs.watch(cssPath, { encoding: 'utf-8' },
                 eventType => {
                     if (eventType == 'change') {
-                        readFile(cssPath).then(newCss => window.customCss.innerHTML = newCss);
+                        readFile(cssPath).then(newCss => {
+                            window.customCss.innerHTML = newCss;
+                            if(window.glasscord)
+                                ipc.send('glasscord_update');
```
Comment (rauenzi) [inline plugins/css_loader.js:64]: This one can be avoided by updating your observer--will DM

Comment (AryToNeX) [reply plugins/css_loader.js]: Yeah, I am dumb

File: plugins/css_loader.js (hunk 1 above)
Comment (AryToNeX) [reply plugins/css_loader.js]: This one will return false 90% of times sadly. I wasn't expecting that.

**Pull request closed without merging**